extensions/fhir/runtime/pom.xml

```diff
@@ -70,11 +70,14 @@
             <groupId>org.jboss.logging</groupId>
             <artifactId>commons-logging-jboss-logging</artifactId>
         </dependency>
-
         <dependency>
             <groupId>com.oracle.substratevm</groupId>
             <artifactId>svm</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ca.uhn.hapi.fhir</groupId>
+            <artifactId>hapi-fhir-server</artifactId>
```

Comment:
As far as I remember the hapi-fhir-server has a dependency on spring-beans: is that required ? if not we can think about excluding it

Reply:
The Spring dependencies are marked as optional so we don't need to exclude them. I've added exclusions for all other dependencies though. Thanks !